tests/console/yast2_lan.pm

```diff
@@ -23,24 +23,6 @@ use version_utils ':VERSION';
 
 my $module_name;
```

Comment:
This is moved to `lib/y2lan_restart_common.pm`, I think this is not needed at all as the module is always `lan`

Reply:
Ah, yeah, right, that was supposed to go away actually; will remove

Reply:
sorry, the variable $module_name is used in the file, please add it back or remove the variable does make sense

Reply:
I actually need to get back to this and fix it again as I accidentally pushed wrong changes into this branch; please disregard this pull request for now until I restore it to the correct state :) I'll get back to this in the afternoon after I tackle other urgent stuff

Reply:
Ok, the mess is fixed; now we can get back to the normal review process :)